chromium_src/components/version_info/version_info.cc

```diff
@@ -8,10 +8,12 @@
 #undef GetChannelString
 #undef GetProductNameAndVersionForUserAgent
 
+#include "brave/version.h"
```

Comment:
per the comment below, the problem here is that we're directly including a generated header when we could use a static header instead and put the generated header inside our own .cc file